lib/web_ui/dev/driver_version.yaml

```diff
@@ -1,4 +1,10 @@
-
+## Version to be picked by this PR. The LUCI will have multiple cipd packages
+## insalled in the recipe at the same time.
+## This mapping will be used to switch between them.
+preferred_version:
+  ## Make sure the major version of the binary in `browser_driver.yaml` is the
+  ## same for Chrome.
+  chrome: '84'
```

Comment:
I think this value should go in the `browser_lock.yaml`. This way when upgrading chrome all you need to do is change one file. Also, when changing `browser_lock.yaml` what will prompt a contributor to also update this file?

Reply:
No it shouldn't.

Soon we would split these such as small tests, large tests. It's good that each test has it's own file.

Reply:
What's the advantage of having its own file? The risk I'm worried about is that we will be running different tests on different versions of the same browser, which usually leads to confusion.

Reply:
done. Also changed the readme.